mitodl/bootcamp-ecommerce
# chore: integrate ruff

### What are the relevant tickets?




Closes https://github.com/mitodl/bootcamp-ecommerce/issues/1485

### Description (What does it do?)


- Removes Pylint, isort, and Black.
- Adds ruff
- Add ruff checks in ci.
- Fixes + Ignore ruff lint errors.
- Fixes ruff formatting.

### How can this be tested?


- Checks should pass
- Verify that new checks are added in CI.
- Nothing should break/change in the App.
